src/app/Dashboard/AddTopic.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './AddTopic.css';
+export default class AddTopic extends React.Component {
+  state = {
+    isHidden: true
+  };
+
+  toggleHidden = () => {
+    this.setState({
+      isHidden: !this.state.isHidden
+    });
+  };
+
+  onSubmit(e) {
+    e.preventDefault();
+    console.log('create new topic with name:', e.target.folderName.value);
+    this.toggleHidden();
+  }
+
+  render() {
+    return (
+      <>
+        <button onClick={() => this.toggleHidden()}>+ add topic</button>
+        {this.state.isHidden ? null : (
```

Comment:
Instead of using a null in a ternary, it might be more idiomatic to write it like `{!this.state.isHidden && {{COMPONENT HERE}}}`. && will keep evaluating statements while they're truthy and return the last one, so that's why this works.